src/rezplugins/shell/gitbash.py

```diff
@@ -2,34 +2,32 @@
 # Copyright Contributors to the Rez Project
 
 
-"""
-Git Bash (for Windows) shell
-"""
+"""Git Bash (for Windows) shell."""
 import os
 import re
 import os.path
 import subprocess
+
 from rez.config import config
 from rezplugins.shell.bash import Bash
 from rez.utils.execution import Popen
 from rez.utils.platform_ import platform_
-from rez.utils.logging_ import print_warning
+from rez.utils.logging_ import print_debug, print_warning
 from rez.util import dedup
 
-if platform_.name == "windows":
-    from ._utils.windows import get_syspaths_from_registry, to_posix_path
+if platform_.name == 'windows':
```

Comment:
Can you undo these changes please? Basically all unnecessary changes in this file. They add noise.